File: leadcrm_import/Leadscrm/Leadscrm/Leads/scratch/search_integration.js
const fs = require('fs');
const path = require('path');

const files = fs.readdirSync('.').filter(f => f.endsWith('.html') || f.endsWith('.js'));

files.forEach(filepath => {
    try {
        const content = fs.readFileSync(filepath, 'utf8');
        const fbMatches = content.match(/facebook|fb_/gi) || [];
        const waMatches = content.match(/whatsapp|wa_/gi) || [];
        if (fbMatches.length > 0 || waMatches.length > 0) {
            console.log(`${filepath}: ${fbMatches.length} FB, ${waMatches.length} WA`);
            const lines = content.split('\n');
            lines.forEach((line, i) => {
                const lower = line.toLowerCase();
                if (lower.includes('facebook') || lower.includes('fb_') || lower.includes('whatsapp') || lower.includes('wa_')) {
                    console.log(`  Line ${i+1}: ${line.trim().slice(0, 120)}`);
                }
            });
        }
    } catch (e) {
        console.error(`Error reading ${filepath}:`, e);
    }
});

File: leadcrm_import/Leadscrm/Leadscrm/Leads/scratch/search_integration.py
import glob
import re

for filepath in glob.glob('*.html') + glob.glob('*.js'):
    try:
        content = open(filepath, encoding='utf-8').read()
        fb_matches = re.findall(r'(?i)facebook|fb_', content)
        wa_matches = re.findall(r'(?i)whatsapp|wa_', content)
        if fb_matches or wa_matches:
            print(f"{filepath}: {len(fb_matches)} FB matches, {len(wa_matches)} WA matches")
            # print matching lines
            lines = content.split('\n')
            for i, line in enumerate(lines):
                if any(x in line.lower() for x in ['facebook', 'fb_', 'whatsapp', 'wa_']):
                    print(f"  Line {i+1}: {line.strip()[:100]}")
    except Exception as e:
        print(f"Error reading {filepath}: {e}")
